tests/Oryx.Integration.Tests/LocalDockerTests/NodeEndToEndTests.cs

```diff
@@ -347,6 +347,75 @@ public async Task CanBuildAndRun_NodeApp(string nodeVersion)
                 });
         }
 
+        [Theory]
+        [InlineData("8.0")]
```

Comment:
we have the class `TestValueGenerator` that can include all these values, perhaps named "AppInsightsNodeVersions`?